docker-compose.yml

```diff
@@ -125,6 +125,24 @@ services:
       - pulp
       - rabbitmq
 
+  task_queue_errata:
+    image: albs-task-queue:latest
+    env_file:
+      - vars.env
+    build:
+      dockerfile: Dockerfile.task-queue
+      context: .
+    volumes:
+      - "./alws:/code/alws"
+    command: bash -c 'source env/bin/activate &&
+                      /wait_for_it.sh rabbitmq:5672 &&
+                      dramatiq alws.dramatiq --threads 1 -Q errata'
```

Comment:
If you want to create a queue as for releases (sequential errata processing) then you need `--processes 1` as well

Reply:
fixed